### ISSUE TITLE
Increasing accuracy by tweaking hyperparameters

### ISSUE THREAD

**Issue by Gituhin** [Gituhin/NLP-Tweet-Classification-using-LSTM#1]
I am getting an accuracy nearly 78% on the validation set, but I want to increase it bit more. Any suggestions or different processes are welcomed.
